README.md

```diff
@@ -703,6 +707,33 @@ A new yml file should be created in `marts/` which contains all tests and docume
         ├── staging.yml
 ```
 
+## Performance
+### Chained View Dependencies
+#### Model
+
+`fct_chained_views_dependencies` ([source](models/marts/performance/fct_chained_views_dependencies.sql)) contains models that are dependent on chains of "non-physically-materialized" models (views and ephemerals), highlighting potential cases for improving performance by switching the materialization of model(s) within the chain to table or incremental. 
+
+This model will raise a `warn` error on a `dbt build` or `dbt test` if the `distance` between a given `parent` and `child` is greater than or equal to 4.
+You can set your own threshold for chained views by overriding the `chained_views_threshold` variable. [See overriding variables section.](#overriding-variables)
+
+#### Example
+
+`table_1` depends on a chain of 4 views (`view_1`, `view_2`, `view_3`, and `view_4`).
+
+<img width="500" alt="dag of chain of 4 views, then a table" src="https://user-images.githubusercontent.com/53586774/176299679-39028eb1-f9e3-492a-bdb7-b72d9d7958b7.png">
+
+#### Reason to Flag
+
+You may experience a long runtime for a model when it is build on top of a long chain of "non-physically-materialized" models (views and ephemerals). Most of this runtime is actually *compilation* time. In the example above, nothing is really computed until you get to `table_1`. At which point, it is going to run the query within `view_4`, which will then have to run the query within `view_3`, which will then have the run the query within `view_2`, which will then have to run the query within `view_1`. These will all be running at the same time, which creates a long runtime for `table_3` and potential storage issues if the data is sufficiently large. 
```

Comment:
I'd be keen to work a bit on the description here. Are we sure that the time is "*compilation*" time and that there might be "storage issues".

Reply:
There doesn't seem to be a ton of dbt documentation on this topic (was pulling from Pat's [notion writeup](https://www.notion.so/dbtlabs/Model-and-Query-Optimization-dea6ec77096349658e38a55553ebbf13)) - feel free to propose a change with a better description!

Reply:
In my mind, making it a bit less specific looks better:

You may experience a long runtime for a model when it is built on top of a long chain of "non-physically-materialized" models (views and ephemerals). In the example above, nothing is really computed until you get to `table_1`. At which point, it is going to run the query within `view_4`, which will then have to run the query within `view_3`, which will then have the run the query within `view_2`, which will then have to run the query within `view_1`. These will all be running at the same time, which creates a long runtime for `table_1`.

Reply:
I like it! I'll make that change :)